site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -34,46 +35,31 @@ export const WorkspacePage: FC = () => {
   const { username, workspace: workspaceName } = useParams() as {
     username: string
     workspace: string
+    template: string
   }
+  const orgId = useOrganizationId()
   const [workspaceState, workspaceSend] = useMachine(workspaceMachine, {
     context: {
+      orgId,
       workspaceName,
       username,
     },
   })
-  const {
-    workspace,
-    getWorkspaceError,
-    getTemplateWarning,
-    getTemplateParametersWarning,
-    checkPermissionsError,
-  } = workspaceState.context
+  const { workspace, error } = workspaceState.context
   const [quotaState] = useMachine(quotaMachine, { context: { username } })
   const { getQuotaError } = quotaState.context
-  const styles = useStyles()
   const failedBuildLogs = useFailedBuildLogs(workspace)
+  const pageError = error ?? getQuotaError
 
   return (
     <RequirePermission
-      isFeatureVisible={getWorkspaceError?.response?.status !== 404}
+      isFeatureVisible={
+        !(isAxiosError(pageError) && pageError.response?.status === 404)
```

Comment:
The logic is still the same or should be 👀. I just had to make the Axios assertion error before because I changed the error to be unknown and TS was complaining about that even when adding the ? operator.

Reply:
My vacation brain saw an `||` instead of an `&&` 😶‍🌫️ - looks good lol